server/views/pages/overview/partials/goalsTab/goalsTabContents.njk

```diff
@@ -90,6 +108,13 @@
                 html: inProgressGoalsHtml
               }
             },
+            {
```

Comment:
I wonder if you've got your feature toggle check either in the wrong place, or it needs to be here as well? 🤔 

As I read the code (and I could be wrong), if the feature toggle is disabled (ie. prod), then the `completedGoalsHtml` variable wont be set (above ☝️ ), BUT this label will still be in the array and therefore a Completed goals tab heading will be rendered, complete with a number (probably zero, but only by virtue of Stephen's code to actually complete a goal not being in prod yet ?). And the it's panel/html property will be set with a variable that hasn't been set yet, which might cause an error?